gradle/libs.versions.toml

```diff
@@ -19,6 +19,8 @@ androidx-compose-ui = "androidx.compose.ui:ui:1.3.0-rc01"
 androidx-compose-material = "androidx.compose.material:material:1.3.0-rc01"
 androidx-compose-material-icons-extended = "androidx.compose.material:material-icons-extended:1.3.0-rc01"
 androidx-compose-constraintlayout = "androidx.constraintlayout:constraintlayout-compose:1.0.1"
+androidx-tooling = "androidx.compose.ui:ui-tooling:1.3.0-rc01"
```

Comment:
I think that it become slightly better to use Gradle version catalogs `[versions]` feature.